xbmc/GUIInfoManager.cpp

```diff
@@ -6715,6 +6715,14 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     @return The parental rating of the list item (PVR).
 ///     <p>
 ///   }
+///   \table_row3{   <b>`ListItem.ParentalRatingCode`</b>,
+///                  \anchor ListItem_ParentalRatingCode
+///                  _string_,
+///     @return The parental rating code (eg: 'PG', etc) of the list item (PVR).
```

Comment:
Please escale the comma or it will break the table (\,).
Ideally try to build doxygen (folder docs/doxygen) and check the output